index.js

```diff
@@ -100,3 +96,35 @@ function proxySignals (child) {
     }
   }
 }
+
+/**
+ * Starts forwarding IPC messages to `child` through `parent`.
+ *
+ * @param parent Parent process.
+ * @param child Child Process.
+ * @return `unproxy` function to stop the forwarding.
+ * @internal
+ */
+function proxyMessages(parent, child) {
+  if (parent.send === undefined) {
+    return noop
+  }
+
+  function childListener(message, handle) {
+    parent.send(message, handle)
+  }
+
+  function parentListener(message, handle) {
+    child.send(message, handle)
+  }
+
+  child.on('message', childListener)
```

Comment:
Closure would have been cleaner in IMO:

```javascript
child.on('message', function (message, handle) {
  child.send(message, handle);
}
```

Same for L122.

Reply:
You can't unproxy it if you do not have a reference to the function.